include/wlr/types/wlr_output_layout.h

```diff
@@ -38,12 +44,28 @@ void wlr_output_layout_remove(struct wlr_output_layout *layout,
  * coordinates relative to the given reference output.
  */
 void wlr_output_layout_output_coords(struct wlr_output_layout *layout,
-		struct wlr_output *reference, int *x, int *y);
+		struct wlr_output *reference, double *x, double *y);
```

Comment:
Why do x and y have to be doubles? within the function it's comparing with struct wlr_output_layout_output x and y fields, which are int.

. . . okay, I see later that everything else got changed to double. I don't understand why yet, but maybe change that in the type as well?